microsoft-teams/1.0/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,60 @@
+= Microsoft Teams Connector 1.0 - Mule 4
+
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]
+
+Anypoint Connector for Microsoft Teams (Microsoft Teams Connector) gives you access to the Microsoft Teams APIs. This connector exposes the most commonly used operations provided by the Microsoft Teams API. 
+
+For compatibility information and fixed issues, see the Microsoft Teams Connector Release Notes.
+
+== Prerequisites
+
+To use this connector, you must be familiar with:
+
+* The Microsoft Teams API
+* Anypoint Connectors
+* Mule runtime engine (Mule)
+* Elements and global elements in a Mule flow
+* Creating a Mule app using Anypoint Studio (Studio)
+
+Before creating an app, you must have:
+
+* Credentials to access the target resource
+* Anypoint Studio version 7.3.5 or later
+
+== Common Use Cases for the Connector
+
+* Notification
++
+Users are notified in Microsoft Teams when a new resource is created or modified in another service, for example, when a new Salesforce case is created.
+* Onboarding
++
+A group of people are hired at the same time and human resources has an email in Gmail or a spreadsheet in Google Sheets that contains information about the new hires. Human resources can use the email or spreadsheet to trigger different actions, for example:
++
+** Add users to a workspace
+** Create a channel for all new hires to instantly share important onboarding information with them
+** Archive the new hire's channel after onboarding is completed
++
+After the onboarding process is completed, resources can be freed or reused for the next onboarding process.
+* Business Teams
++
+** A new team is created for a new business project and the client wants to quickly and automatically integrate this new team with Microsoft Teams.
+** After teams are created, a new employee is hired and the client can automatically executes a series of actions to integrate the new hire with Microsoft Teams.
+** When a team member is either removed or moved to another team, the user can execute a series of automated steps to reflect this change within Microsoft Teams.
+** When the business project ends, the resources allocated for that business project in Microsoft Teams can be reused or removed.
```

Comment:
```suggestion
** When a business project ends, the resources allocated for that business project in Microsoft Teams can be reused.
```
You don't really want to remove the resources, right? They are removed from the ended project, and reused, yes?

Reply:
Either or both. Resources uses resources. In some case, you may not need to reuse them, so to free up resources, you can remove them. Or you can leave the resources and reuse them.